noah/src/main/scala/edu/uci/ics/cloudberry/noah/adm/TweetGeotagNotRequired.java

```diff
@@ -0,0 +1,74 @@
+package edu.uci.ics.cloudberry.noah.adm;
+
+import edu.uci.ics.cloudberry.gnosis.USGeoGnosis;
+import twitter4j.GeoLocation;
+import twitter4j.Status;
+
+public class TweetGeotagNotRequired {
```

Comment:
I was afraid to change things there and something that depends on that would stop working. But, now I think it is ok. Done